### DISCUSSION TITLE
Change default topology view

### DISCUSSION THREAD

**Discussion by Isnubi** [mattieserver/netbox-topology-views#429] [Q&A] [answered]
Hello

I'm facing a problem.
I want to change the default search filter for the topology view but I can't find where I can do that?

Can anyone help me find how to do it ?

Thanks.

**Comment by mattieserver** [comment]
Under the `topology view` menu  you can click the `individual options` sub-menu

**Reply by Isnubi** [reply]
Yes but in these options, I don't have all the options that are available in the filter tab...
Like the possibility to select with the status of the device for example

**Comment by dreng** [comment]
You should use Saved Filters then. That's a NetBox wide concept.

**Reply by Isnubi** [reply]
So I can't change the default filter used by the plugin ?

**Reply by dreng** [reply, marked as answer]
As I wrote before, this is a NetBox wide concept. There is no default filter in any view. To put it anothet way: the default is to have no filter at all. That's why saved filters has been implemented.

To answer your question: no, you can't change the default filter and you can't do that in any other view of NetBox either. Use saved filters.

**Reply by Isnubi** [reply]
Ok thanks
